refinery/data_set_manager/utils.py

```diff
@@ -668,15 +667,21 @@ def generate_solr_params(
 
     if facets_from_config:
         # Twice as many facets as necessary, but easier than the alternative.
-        facet_template = '&facet.field={0}_Characteristics{1}' + \
-                   '&facet.field={0}_Factor_Value{1}'
-        solr_params += ''.join(
-            [facet_template.format(s, NodeIndex.GENERIC_SUFFIX) for s
-             in settings.USER_FILES_FACETS.split(",")]
-        )
-        solr_params += '&fl=*{},name,*_uuid,type,django_id,{}'.format(
-            NodeIndex.GENERIC_SUFFIX, NodeIndex.DOWNLOAD_URL
-        )
+        facet_arr = []
```

Comment:
to utilize the helper methods, I needed to create the facet list first and then create the query. This is the main part of the solution to sync up the file browser filtering with the user files filtering